nodeup/pkg/model/cloudconfig.go

```diff
@@ -99,3 +108,32 @@ func (b *CloudConfigBuilder) Build(c *fi.ModelBuilderContext) error {
 
 	return nil
 }
+
+// We need this for vSphere CloudProvider
```

Comment:
Let's improve this comment a bit. We can add something like- Gets VM UUID, if any, present for this node. This is only required for the vSphere cloud provider.